### ISSUE TITLE
Sequence length: How to limit that to 30, it is increasing automatically as the no. of epochs is increasing 

### ISSUE THREAD

**Issue by pranavraikote** [NVIDIA/vid2vid#160]
I'm custom training vid2vid for Pose-to-body generations, and given below is an extract of the logs used for my custom training.
It says this despite specifying n_frame_total 30 in my parameters used for training ->> --------- Updating training sequence length to 120 ---------

Any way to limit this to 30, or this is the way it has to train for getting good results? Can anyone clarify this? 

----------------Parameters used----------------
TTUR: False
add_face_disc: False
basic_point_only: False
batchSize: 8
beta1: 0.5
checkpoints_dir: ./checkpoints
continue_train: True
dataroot: /mnt/FS/datasets
dataset_mode: pose
debug: False
densepose_only: False
display_freq: 100
display_id: 0
display_winsize: 512
feat_num: 3
fg: False
fg_labels: [26]
fineSize: 256
fp16: False
gan_mode: ls
gpu_ids: [0, 1, 2, 3, 4, 5, 6, 7]
input_nc: 6
isTrain: True
label_feat: False
label_nc: 0
lambda_F: 10.0
lambda_T: 10.0
lambda_feat: 10.0
loadSize: 384
load_features: False
load_pretrain:
local_rank: 0
lr: 0.0002
max_dataset_size: inf
max_frames_backpropagate: 1
max_frames_per_gpu: 5
max_t_step: 1
model: vid2vid
nThreads: 2
n_blocks: 9
n_blocks_local: 3
n_downsample_E: 3
n_downsample_G: 3
n_frames_D: 3
n_frames_G: 3
n_frames_total: 30
n_gpus_gen: 8
n_layers_D: 3
n_local_enhancers: 1
n_scales_spatial: 1
n_scales_temporal: 2
name: /mnt/FS/datasets/vid2vid/test
ndf: 64
nef: 32
netE: simple
netG: composite
ngf: 64
niter: 10
niter_decay: 10
niter_fix_global: 0
niter_step: 5
no_canny_edge: False
no_dist_map: False
no_first_img: False
no_flip: False
no_flow: False
no_ganFeat: False
no_html: False
no_vgg: False
norm: batch
num_D: 2
openpose_only: False
output_nc: 3
phase: train
pool_size: 1
print_freq: 100
random_drop_prob: 0.05
random_scale_points: False
remove_face_labels: False
resize_or_crop: Scaleheight_and_scaledCrop
save_epoch_freq: 1
save_latest_freq: 1000
serial_batches: False
sparse_D: False
tf_log: False
use_instance: False
use_single_G: False
which_epoch: latest
-------------- End ----------------
CustomDatasetDataLoader
dataset [PoseDataset] was created
#training videos = 5070
vid2vid
---------- Networks initialized -------------
-----------------------------------------------
---------- Networks initialized -------------
-----------------------------------------------
Resuming from epoch 14 at iteration 144
update learning rate: 0.000200 -> 0.000140
update learning rate: 0.000200 -> 0.000140
--------- Updating training sequence length to 120 ---------
-------- Updating number of backpropagated frames to 1 ----------
